tools/build_chain.sh

```diff
@@ -1386,14 +1420,14 @@ for line in ${ip_array[*]};do
     echo "Processing IP:${ip} Total:${num} Agency:${agency_array[${server_count}]} Groups:${group_array[server_count]}"
     [ -z "$(get_value ${ip//./}_count)" ] && set_value ${ip//./}_count 0
     for ((i=0;i<num;++i));do
-        echo "Processing IP:${ip} ID:${i} node's key" >> ${logfile}
+        echo "Processing IP:${ip} ID:${i} node's key" >>"${logfile}"
         local node_count="$(get_value ${ip//./}_count)"
         local node_dir="${output_dir}/${ip}/node${node_count}"
         [ -d "${node_dir}" ] && exit_with_clean "${node_dir} exist! Please delete!"
         
         while :
         do
-            gen_cert ${output_dir}/cert/${agency_array[${server_count}]} ${node_dir} "node" >${logfile} 2>&1
+            gen_cert ${output_dir}/cert/${agency_array[${server_count}]} ${node_dir} "node" #>>"${logfile}"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5331982)